webapps-schema/src/main/java/io/camunda/webapps/schema/entities/operation/BatchOperationEntity.java

```diff
@@ -26,6 +26,10 @@ public class BatchOperationEntity extends AbstractExporterEntity<BatchOperationE
 
   @JsonIgnore private Object[] sortValues;
 
+  public BatchOperationEntity() {
+    generateId();
```

Comment:
I feel bad about putting id generation in constructor, as default constructor is also used by Jackson when mapping json data to an object. It must then call setter to set `id` from json, but still I don't feel good about this. (Just a concern, please decide yourself)

Reply:
hmmm good point. I'll try to think of a different way to do it.